server/tools/scenery.js

```diff
@@ -87,9 +87,34 @@ function deleteModel(workspace, body) {
     return true;
 }
 
-function createDefinition(workspace, body) {
-    console.log("Create Definition: " + workspace + ": " + json(body));
-    return false;
+function createDefinition(workspace, body) { 
+    let objects = JSON.parse(fs.readFileSync(root_dir + workspace + '/objects.json'));
+
+
+    const workspacePath = root_dir + workspace
+    const modelPath = "./assets/" + body.pack + "/" + body.model
+    const definitionPath = "/models/definitions/" + body.id.split("-").slice(0,-1).join("/") + "/";
```

Comment:
I like doing:

let pieces = body.id.split('-');
let name = pieces.pop(); // or shift, I get them mixed up
let path = pieces.join('/'); 

This way you get both the filename and folder path.

Reply:
Oh nice didn't think of it